draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -145,13 +145,11 @@
    <date/>
    <abstract><t>
    This document obsoletes RFC8007. The document describes the part of Content Delivery Network
-   Interconnection (CDNI) Control interface that allows a CDN to trigger
-   activity in an interconnected CDN that is configured to deliver
-   content on its behalf. The upstream CDN MAY use this mechanism to
-   request that the downstream CDN preposition metadata or content as well as 
-   request that it invalidate or purge metadata or content. The
-   upstream CDN MAY monitor the status of activity that it has triggered
-   in the downstream CDN.</t>
+   Interconnection (CDNI) Control interface that allows a CDN to trigger activity in an interconnected CDN 
+   that is configured to deliver content on its behalf. 
+   The upstream CDN MAY use this mechanism to request that the downstream CDN preposition metadata or content 
+   and that it invalidate or purge metadata or content. 
```

Comment:
This sentence appears very wordy after the change.  I think this could be simplified to say:
The upstream CDN MAY use this mechanism to request that the downstream CDN preposition, invalidate, and/or purge metadata or content.

Reply:
reads much better
added "metadata and/or content"